conduce/cli.py

```diff
@@ -142,6 +142,9 @@ def find_resource(args):
                 elif not resource.get('mime', 'invalid-mime').startswith('text/'):
                     resource['content'] = base64.b64decode(resource['content'])
 
+    if args.one_line:
+        return list(map((lambda x: "{}: {}".format(x['name'], x['id'])), resources))
```

Comment:
Note that `:` is a valid character in a resource name, so any script using this option will need to look for the last `:` on the line.  This came up because the first thing I tried to search for was resources of type project, and go the following results:
```
[
  "World Cities: d709b9b1-93cb-4192-9e4b-14096977c0e0",
  "DHL Beringe 2: 2017 Jan 31: 0da527ea-d851-4e2f-a673-901d3e5710fe"
]
```